lib/usd/ui/layerEditor/stringResources.h

```diff
@@ -109,7 +109,8 @@ const auto kSaveLayerUsdFileFormatSbm    { create("kSaveLayerUsdFileFormatSbm",
 const auto kBatchSaveAllRelative { create("kBatchSaveAllRelative", "Convert All to Relative Paths.")};
 const auto kBatchSaveRelativeToScene { create("kBatchSaveRelativeToScene", "Relative to Scene File.")};
 const auto kBatchSaveRelativeToParent { create("kBatchSaveRelativeToParent", "Relative to Parent Layer.")};
-const auto kBatchSaveRelativeTooltip { create("kBatchSaveRelativeTooltip", "This path will be relative to ")};
+const auto kBatchSaveRelativeTooltip { create("kBatchSaveRelativeTooltip", "This path will be relative to ^1s.")};
+const auto kBatchSaveRelativeTooltipEnding0 { create("kBatchSaveRelativeTooltipEnding0", "the saved scene file")};
```

Comment:
Sorry but I still think this should be changed to two individual whole strings. This will provide the proper context for the translation team and also allow for proper translation. You cannot just append one string to another as the order of words can change between languages. I think it should be:

```suggestion
const auto kBatchSaveRelativeLayerTooltip { create("kBatchSaveRelativeLayerTooltip", "This path will be relative to the layer: ^1s.")};
const auto kBatchSaveRelativeSceneTooltip { create("kBatchSaveRelativeSceneTooltip", "This path will be relative to the saved scene file.")};
```